src/main/java/org/ihorzima/telegram_notification/bot/TelegramBot.java

```diff
@@ -1,38 +1,58 @@
 package org.ihorzima.telegram_notification.bot;
 
 import lombok.extern.slf4j.Slf4j;
+import org.ihorzima.telegram_notification.config.AdminChatIdHolderConfig;
 import org.ihorzima.telegram_notification.model.Account;
-import org.ihorzima.telegram_notification.repository.AccountRepository;
+import org.ihorzima.telegram_notification.repository.AccountLocalRepository;
+import org.ihorzima.telegram_notification.util.InlineAccountCache;
+import org.springframework.beans.factory.annotation.Value;
 import org.telegram.telegrambots.bots.TelegramLongPollingBot;
 import org.telegram.telegrambots.meta.api.methods.AnswerInlineQuery;
+import org.telegram.telegrambots.meta.api.methods.ParseMode;
 import org.telegram.telegrambots.meta.api.methods.send.SendDocument;
 import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.CallbackQuery;
 import org.telegram.telegrambots.meta.api.objects.InputFile;
+import org.telegram.telegrambots.meta.api.objects.Message;
 import org.telegram.telegrambots.meta.api.objects.Update;
 import org.telegram.telegrambots.meta.api.objects.inlinequery.InlineQuery;
 import org.telegram.telegrambots.meta.api.objects.inlinequery.inputmessagecontent.InputTextMessageContent;
 import org.telegram.telegrambots.meta.api.objects.inlinequery.result.InlineQueryResult;
 import org.telegram.telegrambots.meta.api.objects.inlinequery.result.InlineQueryResultArticle;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.InlineKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.InlineKeyboardButton;
 import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
 
 import java.io.ByteArrayInputStream;
 import java.io.InputStream;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 @Slf4j
 public class TelegramBot extends TelegramLongPollingBot {
+    private static final String ENTER_MANUAL_BUTTON_ID = "btn1";
+    private static final Pattern RECEIVED_LAND_ID_PATTERN = Pattern.compile("(?<=ділянку:\\s?).*");
+    private static final int INLINE_QUERY_LIMIT = 50;
 
-    private final AccountRepository accountRepository;
+    @Value("${bot.user.name}")
```

Comment:
We have `TelegramBotConfig` where we create TelegramBot bean. We can hold `@Value` there and in this class use final variable for both `botUserName` and `adminKeyword` properties